docs/tutorial-create.rst

```diff
@@ -74,50 +74,86 @@ applied on the server.
+Annotate Samples
+================
 
-.. _Resolwe Bioinformatics documentation: http://resolwe-bio.readthedocs.io
-.. _Descriptor schemas: https://resolwe-bio.readthedocs.io/en/latest/descriptor.html
-.. _accompanying examples: https://github.com/genialis/resolwe-bio/tree/master/resolwe_bio/descriptors
+The next thing to do after uploading some data is to annotate samples this data
+belongs to. This can be done by assigning a value to a predefined field on a
+given sample. See example below.
 
-Here, we show how to annotate the ``reads`` data object by defining the
-descriptor information that populates the annotation fields as defined in the
-'reads' descriptor schema:
+Each sample should be assigned a species. This is done by attaching the
+``general.species`` field on a sample and assigning it a value, e.g.
+``Homo sapiens``.
 
 .. literalinclude:: files/tutorial-create.py
-   :lines: 33-42
+   :lines: 33
+
+
+Annotation Fields
+-----------------
 
-We can annotate the sample object using a similar process with a 'sample'
-descriptor schema:
+You might be wondering why the example above requires ``general.species`` string
+instead of e.g. just ``species``. The answer to this are ``AnnotationFields``\ s.
+These are predefined set of names that are available to annotate samples. Let's
+examine one of the fields:
 
 .. literalinclude:: files/tutorial-create.py
-   :lines: 44-59
+   :lines: 35-42
 
-.. warning::
 
-    Many descriptor schemas have required fields with a limited set of choices
-    that may be applied as annotations. For example, the 'species' annotation
-    in a sample descriptor must be selected from the list of options in the
-    `sample descriptor schema`_, represented by its Latin name.
+.. note::
+
+   Each field is uniquely defined by the combination of ``name`` and ``group``.
 
-.. _sample descriptor schema: https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/descriptors/sample.yml
+If you wish to examine what fields are available, use a query
+
+.. literalinclude:: files/tutorial-create.py
+   :lines: 44-46
+
+
+You may be wondering whether you an create your own fields / groups. The answer
+is no. Time has proven that keeping things organized requires usage
+of a selected set of predefined fields. If you absolutely feel that you need an
+additional annotation field, let us know or use resources such as :ref:`metadata`.
+
+
+Annotation Values
+-----------------
+
+As mentioned before, fields are only one part of the annotation. The other part
+are annotation values, stored as a standalone resource ``AnnotationValues``.
+They connect the field with the actual value.
+
+.. literalinclude:: files/tutorial-create.py
+   :lines: 48-55
 
-We can also define descriptors and descriptor schema directly when calling
-``res.run`` function. This is described in the section about the ``run()``
-method below.
+
+As a shortcut, you can get all the ``AnnotationValue``\ s for a given sample by
```

Comment:
For consistency add punctuation mark at the end. In other similar cases there is colon at the end.